content/_data/changelogs/weekly.yml

```diff
@@ -5335,6 +5335,151 @@
+
+  - type: rfe
+    pull: 4349
+    authors:
+      - fqueiruga
+    issue: 59508
+    message: |-
+      Show a tooltip with the full link name when hovering over sidebar links
+
+  - type: rfe
+    pull: 4345
+    authors:
+      - saledouble
+    message: |-
+      Reduce logging level of run completion and update center polling events from INFO to FINEST
+
+  - type: rfe
+    pull: 3983
+    authors:
+      - daniel-beck
+    message: |-
+      Try to always execute flyweight tasks, such as the main builds for Pipeline or Matrix jobs, on the master
+
+  - type: rfe
+    pull: 4347
+    authors:
+      - olamy
+    issue: 60098
+    references:
+      - url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md#54
+        title: full Winstone 5.4 changelog
+      - title: Jetty 9.4.22 changelog
+        url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.22.v20191022
+      - title: Jetty 9.4.21 changelog
+        url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.21.v20190926
+      - title: Jetty 9.4.20 changelog
+        url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.20.v20190813
+      - title: Jetty 9.4.19 changelog
+        url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.19.v20190610
+    message: |-
+      Update Winstone from 5.3 to 5.4 to update Jetty from 9.4.18 to 9.4.22
+
+  - type: rfe
+    pull: 3935
+    authors:
+      - timja
+    issue: 56553
+    message: |-
+      Internal: Make ProxyConfiguration compatible with configuration-as-code plugin.
+      Workaround on the configuration-as-code plugin side is no longer required
+
+  - type: rfe
+    pull: 4330
+    authors:
+      - tsparber
+    message: |-
+      Internal: Remove unused `jenkins-slave.xml` file template from core.
+      Requires WMI Windows Agents plugin version 1.3.1  (Released Mar 14 2017) or newer
+
+  # Bug fixes
+
+  - type: bug
+    pull: 4298
+    authors:
+      - DarphBoBo
+    issue: 59665
+    message: |-
+      Fix `Uninstall` column sorting in the Plugin Manager Install pane
+
+  - type: bug
+    pull: 4342
+    authors:
+      - kdanW
+    issue: 59631
+    message: |-
+      Fix build history table styling
+
+  - type: bug
+    pull: 4346
+    authors:
+      - MRamonLeon
+    issue: 59793
+    message: |-
+      Prevent faulty subtask contributors from leaving builds running forever
+
+  # Developer-facing changes (APIs, extensions, etc.)
+
+  - type: rfe
+    pull: 4350
+    authors:
+      - jglick
+    message: |-
+      Internal: Remove unused remoting library from `jenkins-cli.jar`
+
+  - type: rfe
+    pull: 4351
+    authors:
+      - jglick
+    message: |-
+      Developer: Switch from maven-jenkins-dev-plugin to upstream jetty-maven-plugin
+
+  - type: rfe
+    pull: 4359
+    authors:
+      - Wadeck
+    issue: 60180
+    message: |-
+      Disable HTTP TRACE to prevent security scanner complaints.
```

Comment:
Why do we put it in the bottom?

Reply:
I thought it was the lowest priority for a reader.  It is resolving an inaccurate report from a security scanner by making a code change. It would be better if the security scanners were more accurate, but since the code change is low risk, it was used instead of appealing to the security scanner to make the fix.

That's how I interpreted Wadeck's description